### PR TITLE
minor fixes to punctuation and typos

### PR DESCRIPTION
**Pull request by JohnDoe389**
#### Problem
some typos and confusing punctuation
#### Summary of Changes
if there isn't someone already heavily editing the documentation, i will continue providing these minor corrections...please let me know

Fixes #

### CODE REVIEW

**Comment by jstarry** [comment]
Doc fixes are very welcome but please ensure that lines are wrapped and the format sanity checks pass, thanks!

**Comment by JohnDoe389** [comment]
will do!  wow Justin...super impressed with your speed on getting to this...tx

**Comment by jstarry** [comment]
> will do! wow Justin...super impressed with your speed on getting to this...tx

heh, I'm impressed too. CI is still failing btw

**Comment by JohnDoe389** [comment]
tx...converted the pr to a draft for now

File: docs/src/developing/programming-model/accounts.md [hunk 1]
```diff
@@ -52,10 +51,9 @@ runtime enforces that the account's data (the program) is immutable.
 
 ## Creating
 
-To create an account a client generates a _keypair_ and registers its public key
-using the `SystemProgram::CreateAccount` instruction with preallocated a fixed
-storage size in bytes. The current maximum size of an account's data is 10
-megabytes.
+To create an account, a client generates a _keypair_ and registers its public key
```
Comment (jackcmay) [inline docs/src/developing/programming-model/accounts.md:54]: How about just change "a" to "and" in "preallocated and fixed"?

**Comment by jackcmay** [comment]
@JohnDoe389 You can move it out of draft and besides one nit it looked great.  When it passes CI we'll merge it.

Our CI prevents extra whitespace, looks like that is what is holding up CI currently: https://buildkite.com/solana-labs/solana/builds/48527#96f2c563-99c6-4bc0-98f9-95e35496a772

**Comment by JohnDoe389** [comment]
thank you Jack!...if u can give me an hour i have another commit with that fixed and with improvements to the staking page as well.

**Comment by JohnDoe389** [comment]
looks like i need to check for trailing white space...brb

**Comment by jackcmay** [comment]
@JohnDoe389 Looking good so far, looks like there are still trailing whitspaces

File: docs/src/developing/programming-model/accounts.md (hunk 1 above)
Comment (JohnDoe389) [reply docs/src/developing/programming-model/accounts.md:56]: hi Jack...missed seeing your comment "How about just change "a" to "and" in "preallocated and fixed"...let me go back and look at that again

Comment (jackcmay) [reply docs/src/developing/programming-model/accounts.md:56]: Not sure it is relevant after the latest round of changes :-)

**Review by jackcmay — APPROVED** [review]

**Comment by jackcmay** [comment]
Thanks @JohnDoe389 for the contribution!

**Pull request merged**